tests/dat/actions/timedout.py

```diff
@@ -0,0 +1,3 @@
+def main(dict):
+    while True:
+        0
```

Comment:
in retrospect, i should have used a sleep so as not to churn out cpu load for no reason.